Require null validation check on if statement

Hi,

Today, I was trying to compile my webpack Angular-JS application & found this error message on the log (see the bold text below).

These are the versions of the relevant libraries:
postcss-merge-rules: v2.0.3
node: v5.6.0
css-loader: v0.23.0
cssnano: v3.5.1

After I added 'obj === null' at the if statement, the problem was resolved:
https://github.com/ben-eb/postcss-merge-rules/blob/master/src/lib/clone.js
`````
    if (typeof obj !== 'object' || obj === null) {
        return obj;
    }
`````


Result of:
$ npm run build

`````
> spotz-web-admin@0.0.0 build /Users/farman/spotz-web-admin
> webpack --config webpack.build.js --bail --progress 23% 50/218 build modules

ModuleBuildError: ***Module build failed: TypeError: Cannot read property 'constructor' of null***
    at clone (/Users/farman/spotz-web-admin/node_modules/css-loader/node_modules/cssnano/node_modules/postcss-merge-rules/dist/lib/clone.js:12:25)
    at clone (/Users/farman/spotz-web-admin/node_modules/css-loader/node_modules/cssnano/node_modules/postcss-merge-rules/dist/lib/clone.js:29:25)
    at clone (/Users/farman/spotz-web-admin/node_modules/css-loader/node_modules/cssnano/node_modules/postcss-merge-rules/dist/lib/clone.js:29:25)
    at /Users/farman/spotz-web-admin/node_modules/css-loader/node_modules/cssnano/node_modules/postcss-merge-rules/dist/lib/clone.js:26:24
    at Array.map (native)
    .......
    at Object.<anonymous> (/Users/farman/spotz-web-admin/node_modules/css-loader/lib/loader.js:32:18)

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! spotz-web-admin@0.0.0 build: `webpack --config webpack.build.js --bail --progress`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the spotz-web-admin@0.0.0 build script 'webpack --config webpack.build.js --bail --progress'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the spotz-web-admin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --config webpack.build.js --bail --progress
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs spotz-web-admin
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls spotz-web-admin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/farman/spotz-web-admin/npm-debug.log

--- a/src/lib/clone.js
+++ b/src/lib/clone.js
@@ -3,7 +3,7 @@
 'use strict';
 
 let clone = (obj, parent) => {
-    if (typeof obj !== 'object') {
+    if (typeof obj !== 'object' || obj === null) {
         return obj;
     }
     let cloned = new obj.constructor();

